user_guides/jakartaee/src/main/jbake/content/intro.adoc

```diff
@@ -140,77 +140,122 @@ harness.
 
 For documentation on the test harness used for running the Jakarta EE 10 Platform TCK
 test suite, see
-`https://wiki.openjdk.java.net/display/CodeTools/Documentation`.
+https://wiki.openjdk.java.net/display/CodeTools/Documentation.
 
 [[sthref4]][[jakarta-ee-10-technologies-tested-with-jakarta-ee-10-platform-tck]]
 
-1.2.1 Jakarta EE 10 Technologies Tested with Jakarta EE 10 Platform TCK
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The Jakarta EE 10 Platform TCK test suite includes compatibility tests for the
-following required and optional Jakarta EE 10 technologies:
-
-* Jakarta Authentication 3.0
-* Jakarta Authorization 2.1
-* Jakarta Bean Validation 3.0
-* Jakarta Common Annotations 2.1
-* Jakarta Contexts and Dependency Injection 4.0
-* Jakarta Concurrency 3.0
-* Jakarta Connectors 2.1
-* Jakarta Debugging Support for Other Languages 2.0
-* Jakarta Dependency Injection 2.0
-* Jakarta Enterprise Beans 4.0
-* Jakarta Expression Language 5.0
-* Jakarta Faces 4.0
-* Jakarta JSON Processing 2.1
-* Jakarta JSON Binding 3.0
-* Jakarta Interceptors 2.1
-* Jakarta Mail 2.1
-* Jakarta Messaging 3.1
-* Jakarta Persistence 3.1
-* Jakarta RESTful Web Services 3.1
-* Jakarta Security 3.0
-* Jakarta Server Pages 3.1
-* Jakarta Servlet 6.0
-* Jakarta Standard Tag Library 3.0
-* Jakarta Transactions 2.0
-* Jakarta WebSocket 2.1
-* Jakarta Enterprise Beans 3.2 and earlier entity beans and associated Jakarta Enterprise Beans QL (optional)
-* Jakarta Enterprise Beans 2.x API group (optional)
-* Jakarta Enterprise Web Services 4.0 (optional)
-* Jakarta SOAP with Attachments 3.0 (optional)
-* Jakarta XML Web Services 3.0 (optional)
-
+1.2.1 Jakarta EE 10 Technologies Required for Jakarta EE 10 Platform Compatibility
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Jakarta EE 10 Platform Specification defines the required and optional component specifications.
+The full list with specification version requirements is defined in the Platform EE Specification document (https://jakarta.ee/specifications/platform/10/), see the heading 'Full Jakarta™ EE Product Requirements,'
+
+Jakarta EE 10 TCK tests verify partial compatibility for the Jakarta EE Platform.
+The Platform TCK includes tests for the following components:
```

Comment:
The signature tests validate that the SPEC APIs are 100% correct which does make the above statement about `partial` compatibility verification.

Reply:
I use the word Partial to avoid confusion that running this TCK provides complete compatibility certification. It does not. Compatibility is not verified until this TCK, all rules are followed, and all external TCKs are also run and passing.
The TCK uses functionality from all component technologies that are defined in the specification. Therefore, the first list includes all specifications. The specifications for which this TCK provides compatibility certification are are listed in the table below. If you can suggest different way to make this clear, please offer a suggestion.